package domain.exception;

public class throwUsernameAlreadyExistsException {
    public throwUsernameAlreadyExistsException() throws Exception{
        throw new Exception ("Username already exists.\n");
    }
}
